### ISSUE TITLE
Collision Between Previous and New Versions of Notification Libraries

### ISSUE THREAD

**Issue by raymondTheDev** [SamsidParty/TopNotify#12]
I'm experiencing with the collision between the previous version of your notification library and the new version.
 
Specifically, I have been using the previous version from [SamarthCat/notifications-at-top](https://github.com/SamarthCat/notifications-at-top), and I recently upgraded to the new version from [SamsidParty/TopNotify](https://github.com/SamsidParty/TopNotify). However, I am unable to completely remove the old version, which seems to be causing conflicts.

I have used `TOPLEFT.exe` from the old version, but it does not seem to resolve the issue. Could you please provide instructions on how to manually remove the old version to ensure a smooth transition to the new library?

**Comment by SamsidParty** [comment]
Since the old version is a single .exe file, you have to kill it in task manager and delete the exe file.
After that, restart your PC so explorer and the notification window resets. It should work after you do that.

**Comment by raymondTheDev** [comment]
Upon further investigation, I realized the issue was on my end. I had set up the old version to auto-start on Windows via encrypted registry entries in regedit. After locating and removing those entries, everything is working smoothly now.

I also wanted to express my appreciation for your app. It's been incredibly helpful, and I truly hope Microsoft takes notice and considers implementing this feature natively in Windows. Using a super ultrawide monitor, having notifications appear at the far edge of the screen is quite inconvenient. Your app significantly improves the user experience!  Thank you again!
